docs: reorganise release/version docs (#7033)
typescript-eslint/typescript-eslint: docs/Packages.mdx

---
id: packages
title: Packages
---

> The `@typescript-eslint/*` packages are built from a monorepo located at https://github.com/typescript-eslint/typescript-eslint.

Reviewer: ```suggestion
The `@typescript-eslint/*` packages are all stored in [our GitHub monorepo](https://github.com/typescript-eslint/typescript-eslint).
```

The monorepo is built with [Lerna](https://lerna.js.org) and [Nx](https://nx.dev).

Each page in this section corresponds to a package we intentionally expose to users.
They are:

- [`@typescript-eslint/eslint-plugin`](./packages/ESLint_Plugin.mdx): An ESLint plugin which provides lint rules for TypeScript codebases.

> - [`@typescript-eslint/eslint-plugin-tslint`](./packages/ESLint_Plugin_TSLint.mdx): ESLint plugin that allows running TSLint rules within ESLint to help you migrate from TSLint to ESLint.

Reviewer: ```suggestion
- [`@typescript-eslint/eslint-plugin-tslint`](./packages/ESLint_Plugin_TSLint.mdx): An ESLint plugin that allows running TSLint rules within ESLint to help you migrate from TSLint to ESLint.
```

- [`@typescript-eslint/parser`](./packages/Parser.mdx): An ESLint parser which allows for ESLint to lint TypeScript source code.
- [`@typescript-eslint/scope-manager`](./packages/Scope_Manager.mdx): A fork of [`eslint-scope`](https://github.com/eslint/eslint-scope), enhanced to support TypeScript functionality.

> - [`@typescript-eslint/typescript-estree`](./packages/TypeScript-ESTree.mdx): The underlying code used by [`@typescript-eslint/parser`](./packages/Parser.mdx) that converts TypeScript source code into an <a href="https://github.com/estree/estree">ESTree</a>-compatible form.

Reviewer: Does Markdown link syntax not work here?

Author: 🤷 copy 🍝 I have no idea why this uses an `<a>`
